website/src/controllers/api-v3/development.js

```diff
@@ -4,6 +4,12 @@ import checkForDevelopmentMode from '../../middlewares/api-v3/developmentMode';
 
 let api = {};
 
+api.development = {
```

Comment:
I think naming this as the namespace is good. So, if we change this to debug.js then I think this should be called api.debug.